src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -58,4 +95,68 @@ public String getDescription() {
     public Duration getEstimatedEffortPerOccurrence() {
         return Duration.ofMinutes(5);
     }
+
+    static class RandomCronExpression {
+        private final Random random;
+
+        public RandomCronExpression(Random random) {
+            this.random = random;
+        }
+
+        public int random(int min, int max) {
+            return random.nextInt(max + 1 - min) + min;
+        }
+
+        public int minute() {
+            return random(0, 59);
+        }
+
+        public int hour() {
+            return random(0, 23);
+        }
+
+        public String dayOfWeek() {
+            final List<String> daysOfWeek = Arrays.asList("mon", "tue", "wed", "thu", "fri", "sat", "sun");
+            return daysOfWeek.get(random.nextInt(daysOfWeek.size()));
+        }
+
+        public String dailyCron() {
+            return String.format("%d %d * * *", minute(), hour());
+        }
+
+        public String weeklyCron() {
+            return String.format("%d %d * * %s", minute(), hour(), dayOfWeek());
+        }
+
+        public String weekdays() {
+            return String.format("%d %d * * 1-5", minute(), hour());
+        }
+
+        public String weekends() {
+            return String.format("%d %d * * sat,sun", minute(), hour());
+        }
+
+        public String monthlyCron() {
+
+            return String.format("%d %d %s * *", minute(), hour(), dayOfTheMonth());
+        }
+
+        public int dayOfTheMonth() {
+            return random(1, 28);
+        }
+
+        public String month() {
+            final List<String> months = Arrays.asList("jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sep", "oct", "nov", "dec");
+            return months.get(random.nextInt(months.size()));
+        }
+
+        public String hourlyCron() {
+            return String.format("* %s * * *", hour());
+        }
+
+        public String yearlyCron() {
+            return String.format("%d %d %d %s %s", minute(), hour(), dayOfTheMonth(), month(), dayOfWeek());
```

Comment:
I think we'd best leave out the day of the week here as well.
```suggestion
            return String.format("%d %d %d %s *", minute(), hour(), dayOfTheMonth(), month());
```

Based on [this example](https://crontab.guru/#1_2_3_3_3) it would otherwise seem to schedule both on the third and on every Wednesday.
![image](https://user-images.githubusercontent.com/1027334/223228606-972efbe4-e430-42fd-ad94-99dcfe5bd63b.png)

Versus [the alternative](https://crontab.guru/#1_2_3_3_*).
![image](https://user-images.githubusercontent.com/1027334/223228778-6e6cc3c2-a63f-418a-a7a3-04c3e673ddc8.png)